keyboards/protokeeb/rev1/keyboard.json

```diff
@@ -0,0 +1,507 @@
+                    "label": "MI_E",
+                    "matrix": [1, 0],
+                    "x": 0,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_F",
+                    "matrix": [1, 1],
+                    "x": 1,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_Fs",
+                    "matrix": [1, 2],
+                    "x": 2,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_G",
+                    "matrix": [1, 3],
+                    "x": 3,
+                    "y": 1,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_Gs",
+                    "matrix": [2, 0],
+                    "x": 0,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_A",
+                    "matrix": [2, 1],
+                    "x": 1,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_As",
+                    "matrix": [2, 2],
+                    "x": 2,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_B",
+                    "matrix": [2, 3],
+                    "x": 3,
+                    "y": 2,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MO(3)",
+                    "matrix": [3, 0],
+                    "x": 0,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "TG(1)",
+                    "matrix": [3, 1],
+                    "x": 1,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "TG(2)",
+                    "matrix": [3, 2],
+                    "x": 2,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                },
+                {
+                    "label": "MI_TOGG",
+                    "matrix": [3, 3],
+                    "x": 3,
+                    "y": 3,
+                    "w": 1,
+                    "h": 1
+                }
+            ]
+        }
+    },
+    "audio": {
+        "default": {
+            "on": false,
+            "clicky": false
+        },
+        "macro_beep": false,
+        "pins": ["GP22"],
+        "voices": false
+    },
```

Comment:
```suggestion
    "audio": {
        "default": {
            "on": false
        },
        "pins": ["GP22"],
    },
```

Reply:
Done. Also, the comma ( , ) needs to be removed after the "pins": ["GP22"] line for a valid error-free JSON file.